Documentation/GettingStartedWithTheMRTK.md

```diff
@@ -129,6 +132,19 @@ If you need to move the players start point, then **move the scene content and N
 
 You are now ready to start building your project!
 
+## Recommended build settings (UWP)
+
+When building for WMR, HoloLens 1, or HoloLens 2, it is recommended that your build settings "Target SDK Version"
+and "Minimum Platform Version" look like they do in the picture below:
+
+![](../Documentation/Images/getting_started/BuildWindow.png)
```

Comment:
I can't see this image [here](https://github.com/microsoft/MixedRealityToolkit-Unity/blob/82eb6cb6754dd8f027ba5116ccc15afde52f99f3/Documentation/GettingStartedWithTheMRTK.md#recommended-build-settings-uwp). Is that definitely the right path?

Reply:
Ahhhhhhhh case sensitive paths.